### PR TITLE
Fix custom loop examples to disable train mode during evaluation

### PR DESCRIPTION
**Pull request by bkvogel**
The non-trainer examples (that is, the "custom training loop" examples) previously had a bug that the configuration remained in training mode during test/validation.

### CODE REVIEW

**Comment by okuta** [comment]
How about using `chainer.using_config('enable_backprop')` at the same time?

**Comment by bkvogel** [comment]
I'm not sure I understand. Could you describe the motivation for using `chainer.using_config('enable_backprop')` during test/evaluation mode? Do you mean to disable backprop during test/evaluation?

**Comment by okuta** [comment]
Oh sorry, It mean `chainer.using_config('enable_backprop', False)` .

**Comment by bkvogel** [comment]
Thank you for the suggestion. I have made the change.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
LGTM!
